src/extensions/lens-main-extension.ts

```diff
@@ -25,9 +25,11 @@ import { catalogEntityRegistry } from "../main/catalog";
 import type { CatalogEntity } from "../common/catalog";
 import type { IObservableArray } from "mobx";
 import type { MenuRegistration } from "../main/menu/menu-registration";
+import type { MenuItemConstructorOptions } from "electron";
```

Comment:
I would prefer if we wouldn't expose interfaces from 3rd party libraries in extension API. This limits our ability to adapt to future needs. 

3rd party libraries should always be isolated behind abstractions. 

In this case, we should create own interface which exposes only the features that we want to expose and then in `createTrayMenu` we should map it to the interface that electron requires. That gives us flexibility to do what ever we want in future, even if we choose to implement this menu in some other way.

Reply:
Yeah I thought about. Fair enough. Added a type `TrayMenuRegistration` for that.